dev/com.ibm.ws.jca.utils/resources/com/ibm/ws/jca/utils/internal/resources/JcaUtilsMessages.nlsprops

```diff
@@ -191,5 +191,13 @@ J2CA9942.cfgprop.invalid.parameters.explanation=The annotation @ConfigProperty m
 J2CA9942.cfgprop.invalid.parameters.useraction=Investigate the resource adapter vendor's documentation and problem determination processes.
 
 J2CA9943.missing.connector.dd=J2CA9943E: The resource adapter {0} is invalid. It should contain either a deployment descriptor or a JavaBean class annotated with one of the resource adapter annotations.
-J2CA9943.missing.connector.dd.explanation=A valid resource adapter must have either a deployment descriptor or a JavaBean class annotated with one of @Connector, @AdministeredObject, @ConnectionDefinition. @ConnectionDefinitions or @Activation annotations.
+J2CA9943.missing.connector.dd.explanation=A valid resource adapter must have either a deployment descriptor or a JavaBean class annotated with one of @Connector, @AdministeredObject, @ConnectionDefinition, @ConnectionDefinitions or @Activation annotations.
 J2CA9943.missing.connector.dd.useraction=Investigate the resource adapter vendor's documentation and problem determination processes.
```

Comment:
Investigate the resource adapter vendor's documentation and problem determination processes.

--->

Investigate the vendor documentation and problem determination processes for the resource adapter.